iceoryx_posh/source/roudi/memory/posix_shm_memory_provider.cpp

```diff
@@ -89,7 +87,7 @@ cxx::expected<void*, MemoryProviderError> PosixShmMemoryProvider::createMemory(c
             .hasErrors())
     {
         LogFatal() << "Could not reset signal handler for SIGBUS!";
-        std::terminate();
+        errorHandler(Error::kROUDI_MEMORY__COULD_NOT_UNREGISTER_SIGBUS, nullptr, ErrorLevel::FATAL);
```

Comment:
Please add a return after the `errorHandler`.

Reply:
Done